src/server.cpp

```diff
@@ -2257,6 +2257,20 @@ void Server::fadeSound(s32 handle, float step, float gain)
 		m_playing_sounds.erase(it);
 }
 
+void Server::stopAttachedSounds(u16 id)
+{
+	assert(id);
+
+	for (const auto &sit: m_playing_sounds) {
+		const ServerPlayingSound &sound = sit.second;
+
+		if (sound.object != id)
+			continue;
+
+		stopSound(sit.first);
```

Comment:
the client already knows to do this, so why send him a packet?

Reply:
Will old clients stop the sound on their own?

Reply:
IMO we shouldn't spend effort working around client bugs